apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -171,7 +171,7 @@ export class SpecialAnnouncementActionBlock extends Component {
   render() {
     return (
       <TwoColumnActionBlock
-        imageUrl={pegasus('/images/fill-540x289/teacher-apps.png')}
```

Comment:
Any reason we're keeping this image around still?

Reply:
When it comes to a campaign like this where the images are quite interchangeable, I like the idea of keeping them all around and giving them unique ideas.  It would also let us a/b test or rapidly switch between the images if we went that route.  When we delete the assets at the end we can take them all out at once, which also makes it easier to revive the whole campaign later.

Reply:
In fact, I think we were asked to keep this campaign available to run again next year, so we might end up putting a lot of these assets under DCDO control.